its/projects/DuplicateAnalyzerReferences/FakeGoogleProtobuf/SomeClass.cs

```diff
@@ -0,0 +1,10 @@
+﻿namespace Google.ProtoBuf
```

Comment:
is this the same namespace as google protobuf? ideally it should be the same `Google.Protobuf`

OR make it completely different

IMO we should avoid differentiation by one letter's casing